nlewis84/percussion-library
# Austinchappell/react query

## The Issue

- API requests for ensembles should make use of API filtering to improve performance

## The Solution

- Send filters to API requests
- Paginate the data
- Use react-query to cache responses
- Add a context layer to persist filters

## To Test

- Run `yarn` to install new dependencies
- Run the app
- Go to ensembles, and play around with the filters
- If you select some filters, and then go to "solos", then back to "ensembles", your filters should persist
